src/coreclr/debug/ee/debugger.cpp

```diff
@@ -1129,7 +1129,7 @@ HRESULT Debugger::SetFiberMode(bool isFiberMode)
     CONTRACTL_END;
 
 
-    Thread * pThread = ::GetThread();
+    Thread * pThread = ::GetThreadNULLOk();
```

Comment:
I've tried to not get caught too much with improving code or finding bugs. It is a big change with a potential to rot. But I agree this may be easier to remove. I will take a look.

Reply:
This is indeed dead code. I am removing it. 
I guess anything with a word "fiber" in it should be checked if still needed. Perhaps an idea for another tidy up change.